### DISCUSSION TITLE
제목1, 제목2로 설정할 경우 화면에 노출됩니다.

### DISCUSSION THREAD

**Discussion by 2044smile** [pronist/hELLO#111] [Q&A] [answered]
- 운영체제: windows
- 브라우저: chrome
- 버전: 3.6.1
![image](https://user-images.githubusercontent.com/47213853/174703078-6bd0b9a3-92aa-47f0-a5f0-e916889ce84b.png)

**설명**
최신 버전(3.6.1)을 사용하고 있고, 제목1, 제목2로 설정할 경우 사진과 같이 겹치는 현상이 발생합니다.

**발생 과정**
접속하면 항상 발생합니다. 
화면을 확대하면 보이지 않지만 축소하면 보입니다.
110% 확대 하면 보이지않고, 
100% 확대하면 보입니다.

**스크린샷 및 링크**
![image](https://user-images.githubusercontent.com/47213853/174703176-848cfe8a-8c3c-4509-88cd-f75596da0ac2.png)

**Comment by pronist** [comment, marked as answer]
@2044smile link 태그의 코드를 보니 스킨의 테마가 설정되지 않았습니다. 스킨옵션에 가서 테마를 설정해주세요. 기본적으로 코드를 하나도 건드리지 않았다면 테마가 설정되어 있을텐데 별도로 작업하셨나요?

혹시나해서 덧붙이자면, 만약 README 에 있는대로 티도리 프레임워크에서 스킨을 실행하셨다면 그것은 스킨을 개발하기 위한 실행방법이므로 그럴 의도가 아니었다면 <https://pronist.dev/5> 를 참고하거나 또는 일반적인 티스토리 스킨 설치법을 참고하시면 되겠습니다. 티도리 프레임워크로 개발을 해보실 예정이라면 #104 를 참고하여 현재 `index.xml` 이 티스토리에서 제대로 안 먹는 문제가 있으니 별도로 적용시켜주셔야 한다는 점 참고 부탁드립니다.

**Reply by 2044smile** [reply]
@pronist 
별로도 작업한지 확인이 안되어 다시 재설치하였는데도 같은 현상이 발생했습니다.
ㄴ 스킨테마를 설정해보겠습니다!
